ArduCopter/mode_auto.cpp

```diff
@@ -1210,13 +1383,17 @@ bool ModeAuto::payload_place_run_should_run()
 void ModeAuto::payload_place_run_hover()
 {
     land_run_horizontal_control();
-    land_run_vertical_control(true);
+    // update altitude target and call position controller
+    pos_control->land_at_climb_rate_cm(0.0, false);
+    pos_control->update_z_controller();
 }
 
 void ModeAuto::payload_place_run_descend()
 {
     land_run_horizontal_control();
-    land_run_vertical_control();
+    // update altitude target and call position controller
+    pos_control->land_at_climb_rate_cm(-abs(g.land_speed), true);
```

Comment:
So this is a change in behaviour meaning the vehicle will descend very slowly.  One example is in my S&R competition we have to descend about 30m so that would take a minute with this change while previously it would take less than half this.  I think we should actually default to the WPNAV_SPEED_DN or have a new parameter for the descent speed.

Reply:
I can add a decent rate parameter but I don't think it is needed. When I use it we set a waypoint to the altitude where we want to do the payload place to start to minimise the chance of high release and the associated risks.

Happy to add if you really think it is needed.